test/e2e/framework/test_context.go

```diff
@@ -235,7 +235,7 @@ func RegisterClusterFlags() {
 	flag.StringVar(&cloudConfig.NodeInstanceGroup, "node-instance-group", "", "Name of the managed instance group for nodes. Valid only for gce, gke or aws. If there is more than one group: comma separated list of groups.")
 	flag.StringVar(&cloudConfig.Network, "network", "e2e", "The cloud provider network for this e2e cluster.")
 	flag.IntVar(&cloudConfig.NumNodes, "num-nodes", -1, "Number of nodes in the cluster")
-	flag.StringVar(&cloudConfig.ClusterIPRange, "cluster-ip-range", "10.100.0.0/14", "A CIDR notation IP range from which to assign IPs in the cluster.")
+	flag.StringVar(&cloudConfig.ClusterIPRange, "cluster-ip-range", os.Getenv("CLUSTER_IP_RANGE"), "A CIDR notation IP range from which to assign IPs in the cluster.")
```

Comment:
I'm not a fan of having os.Getenv here.

I would prefer doing it the other way - i.e. having /14 here but overwriting it to content of CLUSTER_IP_RANGE if it is defined.